src/App.js

```diff
@@ -58,7 +63,7 @@ class App extends Component {
       <div className="game">
         <section className="hud">
           <h2 className="score">Litter Spotted: { this.state.points }</h2>
-          <img className="logo" src={logo} alt="Litter Patrol logo" />
+          <img className="logo onClick={() => this.onColorChange('red')}" src={logo} alt="Litter Patrol logo" />
```

Comment:
Hm, I'm not seeing why an `onClick` to the logo was added, or anything about `this.onColorChange` being defined. What's going on here?